image_classifier_3d/proj_trainer.py

```diff
@@ -67,6 +71,7 @@ def run_trainer(self):
 
         trainer = Trainer(
```

Comment:
Following along but had a quick question here: What is the purpose of 'trainer'?